docs/portal.properties-Reference.md

```diff
@@ -181,6 +181,31 @@ installation_map_url=https://installationmap.netlify.app/
 ```
 To set up an installation map instance, one may consult the source code for the installation map [here](https://github.com/cbioportal/installation-map).
 
+## Feature toggles
+These configuration settings are used instead of checking the `app.name` property indirectly.
+```
+# Enable in order to show all cancer studies (default is false)
+show_all_cancer_studies=false
+
+# Enable in order to show the twitter preview (default is false)
+show_twitter=false
+
+# Enable in order to show development channels (default is false)
+show_dev_channels=false
+
+# Enable in order to show the study download list (default is false)
+show_study_download_list=false
+
+# Enable in order to show the reference genome genes (default is false)
+show_reference_genome_genes=false
+
+# Enable if this instance is a private portal (default is false)
+is_private_portal=false
+
+# Enable if this instance is a local DB server (default is false)
+is_localdb_server=false
```

Comment:
these seem very dev related? Not sure if they are feature toggles?

Also `is_private_portal` seems like it might the same as a check on `app.name=="private-portal"`? Is it just used to check if e.g. auth is enabled? If that's the case we should use that instead

I imagine there might also be some banners on MSK portal that are activated like this. For those cases I think it's ok to leave it as is. Ideally we would generalize the config so that you can e.g. do `skin.banner=my banner text`, but that would involve a lot of works, so I'd say out of scope for this PR